e2e_tests/test_darwin.py

```diff
@@ -50,19 +83,78 @@ def test_darwin_create(local_dataset: E2EDataset) -> None:
     assert local_dataset.id is not None
     assert local_dataset.name is not None
 
-
 def test_darwin_push(local_dataset_with_images: E2EDataset) -> None:
     """
     Test pushing a dataset via the darwin cli, dataset created via fixture with images added to object
     """
     assert local_dataset_with_images.id is not None
     assert local_dataset_with_images.name is not None
     assert local_dataset_with_images.directory is not None
+    sleep(2)
     result = run_cli_command(
         f"darwin dataset push {local_dataset_with_images.name} {local_dataset_with_images.directory}"
     )
     assert result[0] == 0
 
+def test_darwin_import(local_dataset_with_annotations: E2EDataset) -> None:
+    """
+    Test importing a dataset via the darwin cli, dataset created via fixture with annotations added to objects
+    """
+    assert local_dataset_with_annotations.id is not None
+    assert local_dataset_with_annotations.name is not None
+    assert local_dataset_with_annotations.directory is not None
+    result = run_cli_command(
```

Comment:
Totally up to you, but you can write these pretty neatly as:

```python
exit_level, std_out, std_err = run_cli_command(
```

and then your asserts become really clear:
```python
assert exit_level == 0
```

Or, in your case here,

```python
exit_level, _, __ = run_cli_command(
```

Your call on this though, I can see why you've gone for this approach too.